### DISCUSSION TITLE
CSC

### DISCUSSION THREAD

**Discussion by NikolaS4yourb** [serenity-is/Serenity#7071] [Help]
Hi,
I try to use CSVexport and have to problem
1.I use šđčćž character, and when export I have another culture character. With excel export is ok.
2.I get this error
![image](https://github.com/serenity-is/Serenity/assets/88085419/93cd2524-1b9a-4d37-8b04-7a9899433cb6)

If someone can help,
Tnx Nikola.

**Comment by volkanceylan** [comment]
You should probably look at docs of https://github.com/JoshClose/CsvHelper. It is not part of Serenity. I think the culture needs to be passed in constructor.
